action.yml

```
@@ -16,6 +16,12 @@ inputs:
     default: 'false'
     description: >-
       Disables validation of the schema of metadata files (devcontainer-feature.json)
+  validate-only:
```

```suggestion
  validate-features-only:
```

Isn't that the input expected in main.ts 🤔 
https://github.com/devcontainers/action/pull/123/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR45

---

yes :)

I was thinking this may be used in the future as a blanket "validate" command - (so could validate templates as well).